lib/main_common.pm

```diff
@@ -1116,14 +1116,22 @@ sub load_console_server_tests {
     }
     if (!is_staging && (is_opensuse || get_var('ADDONS', '') =~ /wsm/ || get_var('SCC_ADDONS', '') =~ /wsm/)) {
         # TODO test on openSUSE https://progress.opensuse.org/issues/31972
```

Comment:
Obsolete TODO now?

Reply:
Not yet, we're working on a new [epic](https://progress.opensuse.org/issues/109569) for a better organization. This PR solves the issue of failing php8 tests only :)

Reply:
You removed the `if is_sle;` though, so the comment is now wrong

Reply:
Comments removed